metadata.rb

```diff
@@ -34,9 +35,11 @@
     suse
     xenserver
     smartos
+    mac_os_x
 }.each do |os|
   supports os
 end
 
+suggests "homebrew"
```

Comment:
So it appears this doesn't work unless homebrew is set to "depends", which is odd bc it's the default provider in Chef 12. Any thoughts on making this "depends" if platform is MacOS?